test/test_combine_wzdx.py

```diff
@@ -0,0 +1,215 @@
+from os import path
+from translator.source_code import combine_wzdx
+from shapely.geometry.polygon import Polygon
```

Comment:
Looks like we're missing `shapely` in the requirements.txt file. When I attempt to run the unit tests, I get:

```
No module named 'shapely'
```

Reply:
should be fixed and also added some of the other in requirement.txt

Reply:
Unit tests are still failing to run:
![image](https://user-images.githubusercontent.com/2932159/120669138-6c644000-c44c-11eb-9cef-7cd821f55742.png)